tests/backup/backup_test.go

```diff
@@ -191,81 +194,153 @@ var _ = Describe("{BasicBackupCreation}", func() {
-			interval_schedule_policy_info := CreateIntervalSchedulePolicy(5, 15, 2)
-			interval_policy_status := Backupschedulepolicy("interval", uuid.New(), orgID, interval_schedule_policy_info)
-			dash.VerifyFatal(interval_policy_status, nil, "Creating interval schedule policy")
+			intervalName := fmt.Sprintf("%s-%s", "interval", timestamp)
+			intervalSchedulePolicyInfo := Inst().Backup.CreateIntervalSchedulePolicy(5, 15, 2)
+			intervalPolicyStatus := Inst().Backup.BackupSchedulePolicy(intervalName, uuid.New(), orgID, intervalSchedulePolicyInfo)
+			dash.VerifyFatal(intervalPolicyStatus, nil, "Creating interval schedule policy")
 
 			log.InfoD("Creating backup daily schedule policy")
-			daily_schedule_policy_info := CreateDailySchedulePolicy(1, "9:00AM", 2)
-			daily_policy_status := Backupschedulepolicy("daily", uuid.New(), orgID, daily_schedule_policy_info)
-			dash.VerifyFatal(daily_policy_status, nil, "Creating daily schedule policy")
+			dailyName := fmt.Sprintf("%s-%s", "daily", timestamp)
+			dailySchedulePolicyInfo := Inst().Backup.CreateDailySchedulePolicy(1, "9:00AM", 2)
+			dailyPolicyStatus := Inst().Backup.BackupSchedulePolicy(dailyName, uuid.New(), orgID, dailySchedulePolicyInfo)
+			dash.VerifyFatal(dailyPolicyStatus, nil, "Creating daily schedule policy")
 
 			log.InfoD("Creating backup weekly schedule policy")
-			weekly_schedule_policy_info := CreateWeeklySchedulePolicy(1, Friday, "9:10AM", 2)
-			weekly_policy_status := Backupschedulepolicy("weekly", uuid.New(), orgID, weekly_schedule_policy_info)
-			dash.VerifyFatal(weekly_policy_status, nil, "Creating weekly schedule policy")
+			weeklyName := fmt.Sprintf("%s-%s", "weekly", timestamp)
+			weeklySchedulePolicyInfo := Inst().Backup.CreateWeeklySchedulePolicy(1, backup.Friday, "9:10AM", 2)
+			weeklyPolicyStatus := Inst().Backup.BackupSchedulePolicy(weeklyName, uuid.New(), orgID, weeklySchedulePolicyInfo)
+			dash.VerifyFatal(weeklyPolicyStatus, nil, "Creating weekly schedule policy")
 
 			log.InfoD("Creating backup monthly schedule policy")
-			monthly_schedule_policy_info := CreateMonthlySchedulePolicy(1, 29, "9:20AM", 2)
-			monthly_policy_status := Backupschedulepolicy("monthly", uuid.New(), orgID, monthly_schedule_policy_info)
-			dash.VerifyFatal(monthly_policy_status, nil, "Creating monthly schedule policy")
+			monthlyName := fmt.Sprintf("%s-%s", "monthly", timestamp)
+			monthlySchedulePolicyInfo := Inst().Backup.CreateMonthlySchedulePolicy(1, 29, "9:20AM", 2)
+			monthlyPolicyStatus := Inst().Backup.BackupSchedulePolicy(monthlyName, uuid.New(), orgID, monthlySchedulePolicyInfo)
+			dash.VerifyFatal(monthlyPolicyStatus, nil, "Creating monthly schedule policy")
 		})
-
 		Step("Register cluster for backup", func() {
-			cluster_status, cluster_uid = RegisterBackupCluster(orgID, "", "")
-			dash.VerifyFatal(cluster_status, api.ClusterInfo_StatusInfo_Online, "Verifying backup cluster")
+			CreateSourceAndDestClusters(orgID, "", "")
```

Comment:
I see CreateSourceAndDestClusters is still using expect, can you update it to dash verifications

Reply:
done!!